docs/verifiable-document/wrapping-document.md

```diff
@@ -0,0 +1,97 @@
+id: wrapping-document
+title: Wrapping Document
+sidebar_label: Wrapping Document
+---
+
+Every OA document has a checksum that provides it a tamper-proof property. At the same time, because the checksum can be used to uniquely identify a document, the checksum (or its derived value) is stored onto the document store as evidence of issuance. To compute the checksum, a `raw document` goes through a process known as `wrapping` to become a `wrapped document`. Only then, the document is ready to be issued onto the blockchain.
+
+Multiple documents can be wrapped at the same time in a single batch operation, creating a single checksum for the entire batch of raw documents. This is especially useful when using document store on the Ethereum blockchain to lower the transaction cost and time.
+
+In this guide, we will learn how to generate the checksum by running the `wrapping` process.
+
+## Pre-requisite
+
+- [OpenAttestation CLI](../component/open-attestation-cli) installed
+- [Understanding the OA Document Schema](/docs/verifiable-document/document-data)
+
+## Wrapping
+
+[In the previous guide](/docs/verifiable-document/document-data) we have learnt how to create one raw document. We will now learn how to use the CLI to wrap it. In this step, we will issue an additional Certificate of Completion to another person at the same time to see how the CLI tool can wrap multiple unwrapped documents at the same time.
+
+### Creating the raw document file
+
+In a directory of your choice, create a folder named `raw-documents`:
+
+```sh
+mkdir raw-documents
+cd raw-documents
+```
+
+Create a file named `coc-1.json` and paste the validated JSON from the previous guide into the file:
+
+`coc-1.json`
+
+```text
+{
+  "name": "OpenAttestation Tutorial Certificate of Completion",
+  "$template": {
+    "name": "COC",
+    "type": "EMBEDDED_RENDERER",
+    "url": "http://localhost:3000"
+  },
+  "recipient": {
+    "name": "John Doe"
+  },
+  "issuers": [
+    {
+      "name": "Demo Issuer",
+      "documentStore": "0x0000000000000000000000000000000000000000",
+      "identityProof": {
+        "type": "DNS-TXT",
+        "location": "demo.openattestation.com"
+      }
+    }
+  ]
+}
+```
+
+Create another file named `coc-2.json` and paste the same validated JSON into the file, changing the `recipient.name` from your name to another person's name.
+
+At this point in time, your directory should look like the following:
+
+```text
+raw-documents
+  |-- coc-1.json
+  |-- coc-2.json
+```
+
+### Wrapping the documents with the CLI tool
+
+Now that we have all the raw documents in a single folder, we will use the CLI tool to read all the files in that folder, wrap documents and then output the files in another directory `wrapped-documents`.
+
+At the same time a `merkle root`, a 64 character long string prepended with `0x` will be generated. The merkle root is the only information that will be stored onto the blockchain to verify the issuance status of an OA document.
```

Comment:
Recommend changing `merkle root` to `merkleRoot` since it directly refers to the JSON key stylized as `merkleRoot`.

Reply:
done